product_hunt_api/product_hunt_api/actions.py

```diff
@@ -0,0 +1,62 @@
+import requests
+import json
+from django.http import HttpResponse
+from .constants import PRODUCT_HUNT_ENDPOINT, HEADERS
+
+
```

Comment:
Approach this by building a class instead.

Reply:
I mean wrap the functions inside a class, instantiate an object of the class with input params and then call the class methods